src/sdb.c

```diff
@@ -115,7 +115,7 @@ static void sdb_fini(Sdb* s, int donull) {
 	free (s->name);
 	free (s->path);
 	ls_free (s->ns);
-	ht_free (s->ht);
+	ht_destroy (s->ht);
```

Comment:
free != destroy

usually free liberates all the fields stored by the object. destroy will just free the object itself without freeing the contents